File: level 36/class/class.py
# N1

def bool_to_word(boolean):
    if boolean:
        return "Yes"
    else:
        return "No"


# N2

def grow(arr):
    result = 1
    for num in arr:
        result *= num
    return result


arr = [1, 2, 3, 4]
print(grow(arr))

File: level 63/class/class.js
// N1

let result = 1;

for (let i = 1; i <= 10000; i++) {
  result *= i;

  if (result > 10000) {
    break;
  }
}

console.log(result);

// N2

for (let i = 1; i <= 200; i++) {
  if (i === 105) {
    break; 
  }
  
  if (i % 3 === 0) {
    continue; 
  }

  console.log(i); 
}

